.travis.yml

```diff
@@ -19,7 +21,12 @@ install:
   - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION pip coverage nose sphinx numpydoc flake8
   - source activate test-env
   - if [[ $DEP_VERSIONS == "oldest" ]]; then
-      conda install numpy==1.7.0 scipy==0.11.0 sympy==0.7.4.1 cython==0.17 theano==0.6.0;
+      conda install numpy==1.8.1 scipy==0.13.3 sympy==0.7.5;
+      if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then
```

Comment:
Could we do the same thing for numpy, scipy, and sympy? i.e. have two sets of mins, one each for python 2 and 3

This is probably just a matter of building some conda binaries for the older versions with Python 3 support, which I can do at a later date.